integration/common/openlineage/common/provider/dbt/processor.py

```diff
@@ -305,6 +307,12 @@ def parse_execution(self, context: DbtRunContext, nodes: Dict) -> DbtEvents:
             if sql:
                 job_facets["sql"] = sql_job.SQLJobFacet(query=sql, dialect=self.extract_dialect())
 
+            run_facets: Dict[str, RunFacet] = {}
+            if tags := output_node.get("tags", None):
+                run_facets["tags"] = tags_run.TagsRunFacet(
+                    tags=[tags_run.TagsRunFacetFields(key=tag, value="true", source="DBT") for tag in tags]
```

Comment:
```suggestion
                    tags=[tags_run.TagsRunFacetFields(key=tag, value=tag, source="DBT") for tag in tags]
```

Imho, when we only have a key not a key:value it's better to duplicate tag in value than to put generic "true". Do you think we can go this way?

Reply:
My thinking is that users can search by value rather than key. So if we have some tag like key=env, value=prod and then equivalent label=prod that we translate to key=prod, value=true, it can be harder to find. I don't feel like adding a "true" out of nowhere to user tags is valuable and wanted from the user perspective - they might rely on value and not key, so they'll have to debug where did the "true" come from, as they did not set it themselves. Maybe that's just preference, feel free to resolve if you don't agree.